BUILD: Requires at least CMake 3.14

### Make sure you've done the following:

- [X] I have read the [installation guide](https://github.com/DarkflameUniverse/DarkflameServer/blob/main/README.md).


### Platform

Ubuntu

### Architecture

x86

### Error Logs

`FetchContent_MakeAvailable` is from CMake 3.14: https://cmake.org/cmake/help/latest/module/FetchContent.html#command:fetchcontent_makeavailable

```
CMake Error at CMakeLists.txt:78 (FetchContent_MakeAvailable):
  Unknown CMake command "FetchContent_MakeAvailable".


-- Configuring incomplete, errors occurred!
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.12)
+cmake_minimum_required(VERSION 3.14)
 project(Darkflame)
 include(CTest)
 

--- a/CMakePresets.json
+++ b/CMakePresets.json
@@ -2,7 +2,7 @@
   "version": 3,
   "cmakeMinimumRequired": {
     "major": 3,
-    "minor": 12,
+    "minor": 14,
     "patch": 0
   },
   "configurePresets": [

--- a/README.md
+++ b/README.md
@@ -46,7 +46,7 @@ If you're using a DLU client you'll have to go into the "CMakeVariables.txt" fil
 ### Linux builds
 Make sure packages like `gcc`, `cmake`, and `zlib` are installed. Depending on the distribution, these packages might already be installed. Note that on systems like Ubuntu, you will need the `zlib1g-dev` package so that the header files are available.
 
-cmake must be version 3.12 or higher!
+CMake must be version 3.14 or higher!
 
 **Build the repository**
 

